ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/pages/participant-view/participant-view.component.html

```diff
@@ -7,30 +7,26 @@
 <div class="govuk-grid-row vh-top-30">
   <div class="govuk-grid-column-one-quarter">
     <app-user-camera-view></app-user-camera-view>
-    <div>
-      <a href="javascript:void()" class="govuk-link" i18n="@@participantView_a">
-      Change camera or microphone
-      </a>
-    </div>
-
   </div>
 </div>
 
 <div class="govuk-grid-row vh-top-30">
   <div class="govuk-grid-column-full">
     <p class="govuk-body" i18n="@@participantView_p">
-      When the video has ended, show the judge's view to see how you will appear to others in the hearing.
+      When the video has ended, click 'Continue'. You can try the practice video hearing again if you like.
     </p>
   </div>
 </div>
 <div class="govuk-grid-row vh-top-30">
   <div class="govuk-grid-column-full">
-    <button type="submit" class="govuk-button govuk-!-margin-right-1" id="continue" (click)="continue()" i18n="@@participantView_btn_showJudgeView">
-      Show judge's view
+    <button type="submit" class="govuk-button govuk-!-margin-right-1" id="continue" (click)="continue()" i18n="@@participantView_btn_continue">
```

Comment:
There's a `journey_continue` or something like that tag I used for the other pages since I figured we might want to keep the same name across pages.

Reply:
Ok

Reply:
I was thinking of this one `@@journey_btn_continue` which doesn't have a page associated with it, otherwise it wouldn't get shared